ui/tests/acceptance/token-test.js

```diff
@@ -856,7 +857,9 @@ module('Acceptance | tokens', function (hooks) {
       await visit('/jobs');
       // Expect the Run button/link to work now
       assert.dom('[data-test-run-job]').hasTagName('a');
-      assert.dom('[data-test-run-job]').hasAttribute('href', '/ui/jobs/run');
+      assert
+        .dom('[data-test-run-job]')
+        .hasAttribute('href', '/ui/jobs/run?namespace=*');
```

Comment:
@philrenaud this is one of the instances where I'm not sure why we need this now (called out in the UI sync doc)

Reply:
Dug into this a bit and I think I have an answer:

[this check](https://github.com/hashicorp/nomad/blob/a9d61ea3fd9ecb51d64534b30fe5938024336787/ui/app/controllers/jobs/index.js#L198-L203) in the jobs index controller sets your namespace to "*" in case your queryParameter has been deleted server-side. So say you had previously bookmarked `/jobs?namespace=myShortlivedNamespace` and it had subsequently been deleted — in order to prevent an empty state, it'd reset you to `*`.

Problem is, the conditional doesn't check for the existence of a namespace queryParam, so it would look to see if `null` was an existing namespace, and since it isn't, it would run the default-namespace-query-param logic for it.

I'm going to add an additional condition to the check and remove the changes to the href/url checks in these tests.